app/src/main/res/menu/menu_play_queue_bg.xml

```diff
@@ -7,4 +7,9 @@
         android:orderInCategory="1999"
```

Comment:
After merging you will probably see this file was renamed. Since the two queues are the same feel free to remove the popup one and to rename this to `menu_play_queue.xml`

Reply:
Yep, this is what I plan to do.